packages/frontend/src/system-integration/notifications.ts

```diff
@@ -260,41 +308,93 @@ async function flushNotifications(accountId: number) {
   )
     .filter(e => e.muted)
     .map(e => e.id)
-  if (mutedChats.length > 0) {
-    // some chats are muted
-    log.debug(`ignoring notifications of ${mutedChats.length} muted chats`)
-  }
-  notifications = notifications.filter(notification => {
-    if (mutedChats.includes(notification.chatId)) {
-      // muted chat
-      log.debug('notification ignored: chat muted', notification)
-      return false
-    } else {
-      return true
-    }
-  })
 
-  if (notifications.length > notificationLimit) {
+  const filteredNotifications = (
+    await Promise.all(
+      notifications.map(async notification => {
+        if (!mutedChats.includes(notification.chatId)) {
+          return notification
+        }
+        // muted chat - only show if it's a mention and mentions are enabled
+        // see  https://github.com/deltachat/interface/pull/78#issuecomment-2536719734
+        if (SettingsStoreInstance.state?.desktopSettings.isMentionsEnabled) {
+          const isMention = await notificationIsMention(accountId, notification)
+          if (isMention) {
+            const chat = await BackendRemote.rpc.getBasicChatInfo(
+              accountId,
+              notification.chatId
+            )
+            if (chat.chatType === C.DC_CHAT_TYPE_GROUP) {
+              // only show mentions for group chats
+              return notification
+            }
+          }
+        }
+        return null
+      })
+    )
+  ).filter(notification => notification !== null)
+
+  if (filteredNotifications.length > notificationLimit) {
     showGroupedNotification(accountId, notifications)
   } else {
     for (const {
       chatId,
       messageId,
-      isWebxdcInfo,
+      notificationType,
       eventText,
-    } of notifications) {
+      contactId,
+    } of filteredNotifications) {
       await showNotification(
         accountId,
         chatId,
         messageId,
-        isWebxdcInfo,
-        eventText
+        notificationType,
+        eventText,
+        contactId
       )
     }
   }
   notificationLimit = NORMAL_LIMIT
 }
 
+/**
+ * if isMentionsEnabled returns true
+ * if the notification is a mention
+ */
+async function notificationIsMention(
```

Comment:
```suggestion
/**
 * returns true if the notification is a mention. See
 * https://github.com/deltachat/deltachat-desktop/issues/4461
 */
async function notificationIsMention(
```

The "if isMentionsEnabled" is no longer relevant.